tests/security/https-overlay/mtls-enabled/https_overlay_test.go

```diff
@@ -0,0 +1,40 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package security
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/istioio"
+)
+
+// TestHTTPSOverlayMtlEnabled simulates the task in docs/tasks/security/authentication/https-overlay/
+// with a sidecar and global mTLS is enabled
+func TestHTTPSOverlayMtlEnabled(t *testing.T) {
```

Comment:
I had to break this out into 2 separate tests due to the docs saying you had to do a re-install. Is there a simpler way?

https://istio.io/docs/tasks/security/authentication/https-overlay/#create-an-https-service-with-istio-sidecar-with-mutual-tls-enabled